Gemfile.lock

```diff
@@ -323,6 +337,7 @@ DEPENDENCIES
   rails_12factor
   redcarpet
   rspec-rails
+  rubocop
```

Comment:
I don't think we need rubocop so, remove it from the Gemfile and undate the Gemfile.lock please